go.mod

```diff
@@ -1,9 +1,8 @@
 module github.com/minio/hperf
 
-go 1.19
+go 1.22.5
```

Comment:
```suggestion
go 1.20
```
.. or are we using newer features? No need to bump this to latest.

Reply:
This is one of those go mod tidy bugs.

Reply:
There are vuln. in older versions and the vuln checker was failing. That's the only reason really.

Reply:
ah then stick it to go 1.21